src/command.h

```diff
@@ -108,6 +108,7 @@ int call_command_no_output(int argc, char** argv)
 int list_commands(int argc, char** argv, Output output);
 int complete_command(int argc, char** argv, Output output);
 
+void try_complete(const char* needle, std::string to_check, Output output);
```

Comment:
This function already existed, it was just not declared in the header. I assume it was okay to add it.

Reply:
Sure